progressbar_example.ipynb

```diff
@@ -0,0 +1,154 @@
+    "        \"\"\"Assign dirichlet to the west and east boundaries. The rest are Neumann by default.\"\"\"\n",
+    "        bounds = self.domain_boundary_sides(sd)\n",
+    "        bc = pp.BoundaryCondition(sd, bounds.west + bounds.east, \"dir\")\n",
+    "        return bc\n",
+    "\n",
+    "    def bc_values_darcy(self, subdomains: list[pp.Grid]):\n",
+    "        \"\"\"Zero bc value on top and bottom, 5 on west side, 2 on east side.\"\"\"\n",
+    "        # Define boundary regions\n",
+    "        values = []\n",
+    "        for sd in subdomains:\n",
+    "            bounds = self.domain_boundary_sides(sd)\n",
+    "            val_loc = np.zeros(sd.num_faces)\n",
+    "            # See section on scaling for explanation of the conversion.\n",
+    "            val_loc[bounds.west] = self.fluid.convert_units(5, \"Pa\")\n",
+    "            val_loc[bounds.east] = self.fluid.convert_units(2, \"Pa\")\n",
+    "            values.append(val_loc)\n",
+    "        return pp.wrap_as_ad_array(np.hstack(values), name=\"bc_values_darcy\")\n",
+    "\n",
+    "\n",
+    "class ModifiedSource:\n",
+    "    def fluid_source(self, subdomains: list[pp.Grid]) -> pp.ad.Operator:\n",
+    "        \"\"\"Assign unitary fracture source\"\"\"\n",
+    "        # Retrieve internal sources (jump in mortar fluxes) from the base class\n",
+    "        internal_sources: pp.ad.Operator = super().fluid_source(subdomains)\n",
+    "\n",
+    "        # Retrieve external (integrated) sources from the exact solution.\n",
+    "        values = []\n",
+    "\n",
+    "        for sd in subdomains:\n",
+    "            if sd.dim == self.mdg.dim_max():\n",
+    "                values.append(np.zeros(sd.num_cells))\n",
+    "            else:\n",
+    "                values.append(np.ones(sd.num_cells))\n",
+    "\n",
+    "        external_sources = pp.wrap_as_ad_array(np.hstack(values))\n",
+    "\n",
+    "        # Add up both contributions\n",
+    "        source = internal_sources + external_sources\n",
+    "        source.set_name(\"fluid sources\")\n",
+    "\n",
+    "        return source\n",
+    "\n",
+    "\n",
+    "class CompressibleFlow(ModifiedGeometry, ModifiedBC, ModifiedSource, SinglePhaseFlow):\n",
+    "    \"\"\"Combining modified geometry, boundary conditions and the source term with the default model.\"\"\"\n",
+    "\n",
+    "    ...\n",
+    "\n",
+    "\n",
+    "fluid_constants = pp.FluidConstants({\"compressibility\": 0.01})\n",
+    "material_constants = {\"fluid\": fluid_constants}\n",
+    "\n",
+    "time_manager = pp.TimeManager(\n",
+    "    schedule=[0, 0.05],\n",
+    "    dt_init=1e-4,\n",
+    "    constant_dt=True,\n",
+    "    iter_max=10,\n",
+    "    print_info=True,\n",
+    ")\n",
+    "\n",
+    "params = {\n",
+    "    \"material_constants\": material_constants,\n",
+    "    \"time_manager\": time_manager,\n",
+    "    \"progressbars\": True,\n",
```

Comment:
Does this notebook (and the corresponding example) illustrate something more than setting `progressbars` to `True`? If not, I am not sure we really need the extra files, and certainly not two files.

Reply:
If the conclusion is that yes, we need an example, It should be possible to illustrate the point with a setup that is much closer to the standard model.